src/parse_frame.cpp

```diff
@@ -84,12 +84,12 @@ void pf_copy(struct parse_frame *dst, const struct parse_frame *src)
  */
 void pf_push(struct parse_frame *pf)
 {
-   static int ref_no = 1;
-
    if (cpd.frame_count < (int)ARRAY_SIZE(cpd.frames))
    {
       pf_copy(&cpd.frames[cpd.frame_count], pf);
       cpd.frame_count++;
+
+      static int ref_no = 1;
```

Comment:
Hmm why is this static in the first place? If there is some deeper meaning behind this, this should not be in the if branch.